### PR TITLE
Add more MNT_ flags, especially on FreeBSD

### PR DESCRIPTION
**Pull request by asomers**

### CODE REVIEW

File: src/unix/bsd/apple/mod.rs [hunk 1]
```diff
@@ -3358,18 +3358,10 @@ pub const DLT_LOOP: ::c_uint = 108;
 pub const BPF_ALIGNMENT: ::c_int = 4;
 
 // sys/mount.h
-pub const MNT_RDONLY: ::c_int = 0x00000001;
```
Comment (devnexen) [inline src/unix/bsd/apple/mod.rs:3361]: just curious any reason removing them ?

Comment (asomers) [reply src/unix/bsd/apple/mod.rs:3361]: I'm not removing them altogether, just moving them into bsd/mod.rs .  Those constants have the same value on all of the BSDs.

Comment (devnexen) [reply src/unix/bsd/apple/mod.rs:3361]: ah right

**Review by JohnTitor — APPROVED** [review]

**Comment by JohnTitor** [comment]
Thanks! @bors r+

**Comment by JohnTitor** [comment]
Ugh: `qemu-system-sparc64: terminating on signal 15 from pid 1533 (timeout)`
@bors retry

**Pull request merged**